gulpfile.js

```diff
@@ -65,19 +63,23 @@ function mapSources() {
     });
 }
 
-gulp.task("min:js", function() {
+gulp.task("min:js", gulp.series(function(done) {
     var tasks = getBundles(regex.js).map(function(bundle) {
         return gulp
-            .src(bundle.inputFiles, { base: "." })
+            .src(bundle.inputFiles, { base: "." , allowEmpty: true})
             .pipe(sourcemaps.init())
             .pipe(mapSources())
             .pipe(concat(bundle.outputFileName))
             .pipe(uglify({ mangle: false }))
             .pipe(sourcemaps.write("."))
             .pipe(gulp.dest("."));
     });
+
+    if(!tasks.length){
```

Comment:
Need to get rid of such callbacks. 
Please try the following:
1. Use merge2 instead of mergeStream for the array (still need to do it sequentiallly)
2. Using promise for our `merge` function (https://gulpjs.com/docs/en/getting-started/async-completion):
```
 return Promise.resolve('the value is ignored');
```

Reply:
used merge2